src/subclass/application.rs

```diff
@@ -0,0 +1,81 @@
+use gtk_sys;
+
+use glib::translate::*;
+
+use glib::subclass::prelude::*;
+
+use Application;
+use ApplicationClass;
+use Window;
+
+pub trait ApplicationImpl: ApplicationImplExt + gio::subclass::prelude::ApplicationImpl + 'static {
+    fn window_added(&self, application: &Application, window: &Window) {
+        self.parent_window_added(application, window)
+    }
+
+    fn window_removed(&self, application: &Application, window: &Window) {
+        self.parent_window_removed(application, window)
+    }
+}
+
+pub trait ApplicationImplExt {
```

Comment:
All the traits should probably get `Gtk` as prefix

Reply:
Done :)